src/Sylius/Bundle/InstallerBundle/Requirement/ExtensionsRequirements.php

```diff
@@ -1,192 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sylius\Bundle\InstallerBundle\Requirement;
-
-use ReflectionExtension;
-use Symfony\Component\Translation\TranslatorInterface;
-
-class ExtensionsRequirements extends RequirementCollection
```

Comment:
We want to support .zip installation too with all vendors/assets compiled, so this is still quite useful check when someone skips composer.